### PR TITLE
First draft OAS for #12

### PR DESCRIPTION
**Pull request by mostlygeek**
I figured it would be easier to discuss #1 if we started with an OAS spec for #12.

### CODE REVIEW

File: api.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+swagger: "2.0"
+info:
+  description: >
+    some description of PollBot
```
Comment (mostlygeek) [inline api.yaml]: should change this before merging ;)

File: api.yaml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+parameters:
+  versionId:
+    name: "versionId"
+    in: "path"
+    description: "version of firefox to check for"
+    required: true
+    type: "string"
+
+paths:
+  /archive/{versionId}:
```
Comment (Natim) [inline api.yaml]: The format should start with the `versionId`, we might also consider adding the `product name`

i.e `/v1/firefox/52.0.1/archive`

File: api.yaml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    url: "irc://irc.mozilla.org:6696/product-delivery"
+  license:
+    name: "Mozilla Public License 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "pollbot.services.mozilla.com"
+basePath: "/v1"
+schemes:
+- "https"
+
+parameters:
```
Comment (Natim) [inline api.yaml]: Let's add a product name.

**Comment by Natim** [comment]
- [ ] add a test to make sure the definition is valid

**Pull request merged**